src/Microsoft.VisualStudio.Editors/OptionPages/GeneralOptionPage.vb

```diff
@@ -114,7 +114,7 @@ Namespace Microsoft.VisualStudio.Editors.OptionPages
             ' The main copy of the options has changed. Update the copy used by the
             ' Tools | Options UI, if has been created.
             If _shouldUpdateOptionsControlOnPropertyChange AndAlso _optionsControl IsNot Nothing Then
-                ThreadHelper.JoinableTaskFactory.RunAsync(
+                Dim unused = ThreadHelper.JoinableTaskFactory.RunAsync(
```

Comment:
why is the variable needed?

Reply:
This is the VB equivalent of a discard in C# (i.e. `_ = ThreadHelper...`).

In this case `RunAsync` returns a `Task` that is never being awaited. The newer version of the threading analyzers flags this as a warning, as we generally want to await tasks. In cases where we don't wish to await them, we use a discard like this to hint to other developers (and the analyzer) that the task is intentionally being let free without being awaited.

Reply:
Likely a warning about the result of it not being assigned. We have `_ = ` in C# for it. I don't think there is an equivalent in VB.

Reply:
today I learned 🙂 thanks!

Reply:
https://github.com/dotnet/vblang/issues/346